docs/source/python/filesystems.rst

```diff
@@ -178,11 +178,19 @@ Example how you can read contents from a S3 bucket::
    >>> f.readall()
    b'some data'
 
+   # Note, to resolve the S3 bucket, one can use `resolve_s3_region('my-test-bucket')`
+   >>> s3 = fs.S3FileSystem(region=resolve_s3_region('my-test-bucket'))
+
+   # Or alternatively...
+   >>> s3 = fs.S3FileSystem.from_uri('s3://[access_key:secret_key@]bucket/path[?region=]')
```

Comment:
```suggestion
   >>> s3, path = fs.S3FileSystem.from_uri('s3://[access_key:secret_key@]bucket/path[?region=]')
```

`from_uri` returns two elements. 

Also, maybe leave out the `[?region]` part, since the goal here is to show that `from_uri` will infer the region from the bucket name (without the need to manually specify it)